Dockerfile.build

```diff
@@ -0,0 +1,23 @@
+FROM node:10
```

Comment:
afaik we will also get syntax highlighting by rename this to `Build.Dockerfile` or `build.Dockerfile`